src/pages/billing/NewAccountModal.js

```diff
@@ -330,6 +330,7 @@ const NewAccountModal = ({ onDismiss }) => {
         h(ButtonSecondary, { onClick: () => setPage(page - 1), disabled: onFirstPage }, ['Back']),
         h(ButtonPrimary, {
           style: { marginLeft: '2rem' },
+          // eslint-disable-next-line no-console
```

Comment:
Congrats, you found dead code! Please just delete this file.

Reply:
I was wondering about that "TODO: add submit action here", but I didn't look too closely. Deleted in https://github.com/DataBiosphere/terra-ui/pull/2940/commits/ef5c5859ce1280f870cb5892831fd32e1fb19cb4.